src/peer_connection/certificate.rs

```diff
@@ -168,7 +173,24 @@ impl RTCCertificate {
     func CertificateFromX509(privateKey crypto.PrivateKey, certificate *x509.Certificate) Certificate {
         return Certificate{privateKey, certificate, fmt.Sprintf("certificate-%d", time.Now().UnixNano())}
     }
+    */
+
+    pub(crate) async fn collect_stats(
+        &self,
+        collector: &Arc<Mutex<StatsCollector>>,
+        worker: Worker,
+    ) {
+        let fingerprints = self.get_fingerprints().unwrap();
+        if let Some(fingerprint) = fingerprints.into_iter().nth(0) {
+            let stats = CertificateStats::new(self, fingerprint);
+            let mut lock = collector.try_lock().unwrap();
+            lock.push(StatsReportType::CertificateStats(stats));
+
+            drop(worker);
+        }
```

Comment:
Pion gets `base64_certificate` and `issuer_certificate_id`, with some serialization. I don't see the actual source data for those in webrtc.rs.